parser/src/main/scala/jawn/Parser.scala

```diff
@@ -89,7 +89,8 @@ abstract class Parser[J] {
   protected[this] def column(i: Int): Int
 
   protected[this] final val HexChars: Array[Int] = {
```

Comment:
Also not a big deal, and not related to this PR, but is there a reason `HexChars` is an instance val? It seems unnecessary to instantiate this array for every string parsed.

Reply:
Sure! I moved all instance vals into the `Parser` companion except for `utf8` which is used in many of the subclasses. (I could move that one too but it seemed OK to leave it.)